build.rs

```diff
@@ -0,0 +1,21 @@
+#[macro_use]
+extern crate clap;
+
+use clap::Shell;
+
+include!("src/clap_app.rs");
+
+const BIN_NAME: &str = "bat";
+
+fn main() {
+
+    let outdir = match std::env::var_os("OUT_DIR") {
```

Comment:
Can we use the same mechanism as in `fd` here:
```rust
let outdir = std::env::var_os("SHELL_COMPLETIONS_DIR").or(std::env::var_os("OUT_DIR"));
```

See https://github.com/sharkdp/fd/issues/121


It would also require us to use `fs::create_dir_all`:
https://github.com/sharkdp/fd/commit/3104729f29f4c1bb3a49b7adc2fa79f355115fd8